### ISSUE TITLE
Request to add uninstall option in Makefile

### ISSUE THREAD

**Issue by BvdLind** [meskarune/i3lock-fancy#151]
It would be really handy to have an uninstall option in the Makefile, instead of having to remove everything manually.

**Comment by jeanrafaellourenco** [comment]
@BvdLind maybe it can be solved this way, in the Makefile add:

```bash
uninstall:
	@unlink $(BINDIR)/$(PRGM) # Remove binary
	@rm -Rf $(SHRDIR)/$(PRGM) # Remove icons
	@unlink $(SHRDIR)/man/man1/i3lock-fancy.1 # Remove man
	@rm -Rf $(SHRDIR)/licenses/$(PRGM) # Remove license

**Comment by AladW** [comment]
PR?

**Comment by jeanrafaellourenco** [comment]
> PR?

https://github.com/meskarune/i3lock-fancy/pull/166

**Comment by AladW** [comment]
Merged #166
